values-observer-scs.yaml

```diff
@@ -28,13 +28,15 @@ thanos:
 #        http: 30002
     stores:
       - dnssrv+_grpc._tcp.kube-prometheus-thanos-ruler
-## FIXME: Thanos query in Harbor cluster is temporary exposed via (unsecure) NodePort, see https://github.com/SovereignCloudStack/k8s-observability/issues/32
+## FIXME: Thanos query in Harbor and Moin cluster is temporary exposed via (unsecure) NodePort, see https://github.com/SovereignCloudStack/k8s-observability/issues/32
 ##   Thanos query should be exposed via ingress and then the store address below should be used.
 ##      - dnssrv+_http-harbor-cluster._tcp.thanos-query-envoy
 ## Uncomment this if you want to observe Harbor cluster via node port
       - 213.131.230.86:31050
-## FIXME: Thanos query in Harbor cluster is temporary exposed via (unsecure) NodePort, see https://github.com/SovereignCloudStack/k8s-observability/issues/32
-##   Thanos query in Harbor cluster should be exposed via ingress and then the section below should be used.
+## Uncomment this if you want to observe Moin cluster via node port
+      - 213.131.230.199:31050
```

Comment:
If you want to be precise with the comments, I suggest adding a similar entry `dnssrv+`... for the moin cluster as a comment.
You also added the `thanosQueryEnvoySidecar.config` item on line 108. IMO also the `sidecars` and `extraVolumes` sections should be modified.

Reply:
Yes, miss that, thanks. But Why do you want to modify `extraVolumes`?

Reply:
`extraVolumes` are probably not needed, sorry.